tests/templates/SSViewerTestIncludeScopeInheritanceInclude.ss

```diff
@@ -1 +1 @@
-$Title <% if ArgA %>- $ArgA <% end_if %>- <%if First %>First-<% end_if %><% if Last %>Last-<% end_if %><%if MultipleOf(2) %>EVEN<% else %>ODD<% end_if %> top:$Top.Title
+<% if $Title %>$Title<% else %>Untitled<% end_if %> <% if ArgA %>_ $ArgA <% end_if %>- <%if First %>First-<% end_if %><% if Last %>Last-<% end_if %><%if MultipleOf(2) %>EVEN<% else %>ODD<% end_if %> top:$Top.Title
```

Comment:
I'm really surprised this doesn't throw themplate parse errors.

`<%if ...` (no space between `%` and `if` always caused problems for me.

Please can you update this template to use `$` prepended vars (eg: `if $First`)